fs/xfstests.py

```diff
@@ -221,6 +229,90 @@ def setUp(self):
         self.test_mnt = self.params.get('test_mnt', default='/mnt/test')
         self.disk_mnt = self.params.get('disk_mnt', default='/mnt/loop_device')
         self.fs_to_test = self.params.get('fs', default='ext4')
+        self.run_type = self.params.get('run_type', default='distro')
+
+        if self.run_type == 'upstream':
+            prefix = "/usr/local"
+            bin_prefix = "/usr/local/bin"
+            sbin_prefix = "/usr/local/bin"
```

Comment:
Did you actually mean /usr/local/bin and /usr/local/sbin ? If not then just one variable would suffice.